tests/modules/notebooks/test_sagemaker_notebook.py

```diff
@@ -11,7 +11,7 @@ def stop_instance(self):
     def get_notebook_instance_status(self):
         return "INSERVICE"
 
-
+# TODO: question, why do we need the org1 and env1 fixtures?
```

Comment:
Seems that we don't need them. They are leftovers

Reply:
I'll remove them then